src/style.css

```diff
@@ -1 +1,45 @@
-/* add styles here */
+:root {
+  margin: 0;
+}
```

Comment:
Don't use the `:root` selector to set margins. The `:root` selector is intended for defining custom properties (CSS variables). If you want to reset the margin for the entire page, it's common to set `margin: 0;` on the `body` element instead.